docassemble/GithubFeedbackForm/data/questions/browse_feedback_sessions.yml

```diff
@@ -60,20 +86,25 @@ if: ids
 id: main browse screen
 question: |
   Select feedback to browse
+subquestion: |
+  There are two main types of feedback:
+
+  * feedback attached to Github Issues
+  * feedback not attached to Github Issues (Reviews)
 fields:
   - What type of feedback: feedback_type
```

Comment:
This is still pretty clunky but I know it's out of scope for this PR.

Reply:
Totally fair: I'll try to revive https://github.com/SuffolkLITLab/docassemble-GithubFeedbackForm/issues/28 with the progress so far after merging to discuss what we should change and how.

Reply:
I'm thinking more that we show all kinds of feedback on one screen; the mode switching is not super intuitive. But it's possible we need modes; if that's the case, maybe a section/tab metaphor would be better.

Reply:
Thanks. TBH, I was getting confused when trying to test this; how I split it up (and the names, ugh) weren't great. I agree, one screen makes a lot more sense, so I cleaned up a bit of code to do that.

At some point, there'll need to be some way to handle lots of feedback, but we'd run into usage issues (there are 100s of pieces of feedback on this server, I can't find new ones!) before we run into performance issues, and the old way of doing things wasn't going to deal with the usage issues either.